refinery/ui/source/js/user-file-browser/services/factory.js

```diff
@@ -68,8 +68,13 @@
                   '</a>' +
                 '</div>',
             width: 30 }];
+      // temp solution to handle empty columns until backend is updated to
+      // avoid sending attributes with no fields
+      var validateAttributeStr = culledAttributes.join('');
       settings.djangoApp.userFilesColumns.forEach(function (column) {
-        defs.push({ field: column });
+        if (validateAttributeStr.indexOf(column) > -1) {
```

Comment:
I'd worry about getting false positives from chance concatenations... Could you use the array instead?
```javascript
_.contains(culledAttributes, column)
```
(or a built-in, if the JS version we're targetting supports it.)

Reply:
@mccalluc I need to make a few other changes also, but yes your suggestion makes sense.